korge/src/commonMain/kotlin/korlibs/korge/view/CachedContainer.kt

```diff
@@ -36,7 +36,9 @@ open class FixedSizeCachedContainer(
 
 open class CachedContainer(
     @property:ViewProperty
-    var cache: Boolean = true
+    var cache: Boolean = true,
+    @property:ViewProperty
+    var expensiveScaling: Boolean = true,
```

Comment:
Can we have `expensiveScaling: Boolean? = null` ?

When constructing KorGE we have a `quality` parameter. Can we use the quality parameter to determine the value when not specified?

Here's the quality:
https://github.com/korlibs/korge/blob/b9213f1adc4d66c2148fca9363ca520874a1a696/korgw/src/commonMain/kotlin/korlibs/render/GameWindow.kt#L480

Reply:
This PR https://github.com/korlibs/korge/pull/1762/files will be needed, so it is possible to access tue GameWindow quality inside the RenderContext

Reply:
> Can we have `expensiveScaling: Boolean? = null` ?

Excellent idea, implemented! https://github.com/korlibs/korge/pull/1760/commits/59cb48385fc87c92190146f4d759a8187d3cdd5e